src/main/java/com/revature/g2g/models/SkillDTO.java

```diff
@@ -0,0 +1,14 @@
+package com.revature.g2g.models;
+
+public class SkillDTO {
+	private long skillId;
+	private String name;
+	private SkillDTO parentSkill;
+
+	public SkillDTO(Skill skill) {
+		super();
+		this.skillId = skill.getSkillId();
+		this.name = skill.getName();
+		this.parentSkill = new SkillDTO(skill.getParentSkill() );
+	}
```

Comment:
We don't want this to infinitely loop if they create a loop of skills. Maybe overload with a limiter?
`public SkillDTO(Skill skill, boolean stopParentSkill){
    super();
    this.skillId = skill.getSkillId();
    this.name = skill.getName();
    if (stopParentSkill){
        this.parentSkill = new SkillDTO(skill.getParentSkill(), false);
    } else{
        this.parentSkill = null;
    }
}`